### DISCUSSION TITLE
生物无法正常掉落物品 Dropping loot from non-player death source

### DISCUSSION THREAD

**Discussion by EdmYouth** [Rad586/Forget-Me-Not#103] [Q&A] [answered]
### Descripiton 简述

2.3.0版本。生物被非玩家攻击性致死后无法掉落物品。比如生电中的营火、灵魂营火、岩浆等处死生物后没有掉落物。请问如何修改。

### Steps to reproduce. 复现问题的步骤

如何在使用其他方式处死生物时正常掉落物品

**Comment by Rad586** [comment, marked as answer]
已移至讨论区，整合包设计不属于 bug。
Moved to Discussions since it's intended.

因为会出一些战利品上的问题，不推荐这么做，但如果你无论如何都想做：
删除 `.minecraft\config\openloader\data\z_fmn_tweaks\data\minecraft\loot_tables\entities` 下的所有文件。
I won't recommend doing this since it will break some stuffs, but if you want to do it anyway: 
Remove all files in `.minecraft\config\openloader\data\z_fmn_tweaks\data\minecraft\loot_tables\entities`

请注意，2.3.0 并不是最新版本，版本特定问题不会获得支持。
Please notice that 2.3.0 is depreciated, issues specific to that version won't be supported.
